src/main/scala/cromwell/engine/db/slick/SymbolComponent.scala

```diff
@@ -0,0 +1,54 @@
+package cromwell.engine.db.slick
+
+case class Symbol
+(
+  workflowExecutionId: Int,
+  scope: String,
+  name: String,
+  iteration: Option[Int],
+  io: String,
+  wdlType: String,
+  wdlValue: String,
+  symbolId: Option[Int] = None
+  )
+
+trait SymbolComponent {
+  this: DriverComponent with WorkflowExecutionComponent =>
+
+  import driver.api._
+
+  class Symbols(tag: Tag) extends Table[Symbol](tag, "SYMBOL") {
+    def symbolId = column[Int]("SYMBOL_ID", O.PrimaryKey, O.AutoInc)
+
+    def workflowExecutionId = column[Int]("WORKFLOW_EXECUTION_ID")
+
+    def scope = column[String]("SCOPE")
+
+    def name = column[String]("NAME")
+
+    def iteration = column[Int]("ITERATION")
```

Comment:
Careful if you take my other suggestion about allowing nulls for iteration that this might need to be changed to a non-primitive that will allow nulls, or maybe the magic here takes care of that automatically?